src/core/qgismobileapp.h

```diff
@@ -57,6 +57,10 @@ class QgsProject;
 
 #define REGISTER_SINGLETON(uri, _class, name) qmlRegisterSingletonType<_class>( uri, 1, 0, name, [] ( QQmlEngine *engine, QJSEngine *scriptEngine ) -> QObject * { Q_UNUSED(engine); Q_UNUSED(scriptEngine); return new _class(); } )
 
+#define SUPPORTED_PROJECT_EXTENSIONS QStringList( { QStringLiteral( "qgs" ), QStringLiteral( "qgz" ) } )
+#define SUPPORTED_VECTOR_EXTENSIONS    QStringList( { QStringLiteral( "gpkg" ), QStringLiteral( "shp" ), QStringLiteral( "kml" ), QStringLiteral( "kmz" ), QStringLiteral( "geojson" ), QStringLiteral( "json" ), QStringLiteral( "pdf" ) } )
+#define SUPPORTED_RASTER_EXTENSIONS    QStringList( { QStringLiteral( "tif" ), QStringLiteral( "pdf" ), QStringLiteral( "jpg" ), QStringLiteral( "png" ), QStringLiteral( "gpkg" ) } )
+
```

Comment:
```suggestion
#define SUPPORTED_PROJECT_EXTENSIONS QStringList( { 
  QStringLiteral( "qgs" ),
  QStringLiteral( "qgz" ),
} )
#define SUPPORTED_VECTOR_EXTENSIONS QStringList( {
  QStringLiteral( "gpkg" ),
  QStringLiteral( "shp" ),
  QStringLiteral( "kml" ),
  QStringLiteral( "kmz" ),
  QStringLiteral( "geojson" ),
  QStringLiteral( "json" ),
  QStringLiteral( "pdf" ),
} )
#define SUPPORTED_RASTER_EXTENSIONS QStringList( {
  QStringLiteral( "tif" ),
  QStringLiteral( "pdf" ),
  QStringLiteral( "jpg" ),
  QStringLiteral( "png" ),
  QStringLiteral( "gpkg" ),
 } )
```
Much easier to spot what is supported and what isn't at first glance.

Reply:
I'm pretty sure it won't compile here, #define can't spill over multiple lines if I'm not wrong. Let me try.

Reply:
![image](https://user-images.githubusercontent.com/1728657/106359720-099c5400-6347-11eb-86f1-311f641d46a9.png)

Reply:
aahh, okay :( no worries. The other option is to add it as static to `QgisMobileApp` itself, leave it to your personal taste.